.github/workflows/test-pr.yml

```diff
@@ -180,3 +180,49 @@ jobs:
         if: always()
         run: |
           docker stop --time=0 ${CONTAINER_NAME}
+
+  nix:
+    needs: code-quality-checks
+    name: 'Nix Tests'
+    strategy:
+      fail-fast: false
+      matrix:
+        runner: [ubuntu-latest, macos-13]
```

Comment:
we should be testing this on our m1 machines as well, since we want to set up the cachix cache next for fast kontrol installs via kup

Reply:
Done.